lib/haml_lint/tree/tag_node.rb

```diff
@@ -198,5 +206,17 @@ def script
     def text
       (@value[:value] if @value[:parse]) || ''
     end
+
+    private
+
+    def parsed_attributes
+      HamlLint::RubyParser.new.parse(hash_attributes_source)
```

Comment:
Here's the spike I was talking about. Do you think this a better approach? https://github.com/ckundo/haml-lint/commit/f74eaf64b02468a3f3bef7f465504d9d0bd889bb